### DISCUSSION TITLE
app 内有个法国节点

### DISCUSSION THREAD

**Discussion by feiyu02** [eycorsican/leaf#228] [General]
app 内有个法国节点

**Comment by anwenzen** [comment]
- 这都被你发现了，我就说一直配置vmess不成功，没想到里面有个节点，覆盖了我配置的文件，看了评论还说不支持vmess🤣

**Reply by tc2024** [reply]
你好 请问下 vmess怎么配置  我这边尝试了很多方案都不行  其他的协议都可以

**Reply by anwenzen** [reply]
就是不支持VMess了，所以换协议吧

**Reply by tc2024** [reply]
谢谢，历史版本里 是不是有支持的VMESS的版本

**Reply by anwenzen** [reply]
> 谢谢，历史版本里 是不是有支持的VMESS的版本

你可以试试
